sycl/source/detail/program_manager/program_manager.cpp

```diff
@@ -136,13 +136,13 @@ static RT::PiProgram createSpirvProgram(const ContextImplPtr Context,
 
 RTDeviceBinaryImage &
 ProgramManager::getDeviceImage(OSModuleHandle M, const string_class &KernelName,
-                               const context &Context) {
+                               const context &Context, bool FromProgramApi) {
```

Comment:
Seems like my prior comment got lost - I suggested that the name of this parameter resembles its semantics (how it is supposed to change getDeviceImage bahavior) rather than the caller classification.

Reply:
Thanks, I renamed this variable.